src/partials/auth-security.md

```diff
@@ -34,3 +34,18 @@ Enable email alerts for your users so that whenever another session is created f
 You won't receive notifications when logging in using [Magic URL](/docs/products/auth/magic-url), [Email OTP](/docs/products/auth/email-otp), or [OAuth2](/docs/products/auth/oauth2) since these authentication methods already verify user access to their systems, establishing the authentication's legitimacy.
 
 To toggle session alerts, navigate to **Auth** > **Security** > **Session alerts**.
+
+# Memberships privacy {% #memberships-privacy %}
+
+Hide specific membership details from other users by marking them as private. To set this up, go to **Auth** > **Security** > **Memberships privacy**
+
+These details can be made private:
+
+- `userName` - The member's name
+- `userEmail` - The member's email address
+- `mfa` - Whether the member has enabled multi-factor authentication
+
+{% info title="Scale plan feature" %}
+Memberships privacy is available on the Appwrite Cloud scale plan.
+{% /info %}
```

Comment:
```suggestion
```

We don't need this anymore since we're going to make this feature available to all plans.